src/excalidraw-utils/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
is this needed?

Instead, we can add instructions on how to import and use it in the readme of this package?

Reply:
Not really, this file is only used for local/manual testing.
We should definitely add a README and we can remove this file if needed.